modules/stats.php

```diff
@@ -70,6 +73,33 @@ function stats_enqueue_dashboard_head() {
 }
 
 /**
+ * Checks do_not_track config.
+ *
+ * @return true if config honors DNT, false if not.
+ */
+function stats_honor_dnt_header_for_stats() {
+	$options = stats_get_options();
+
+	if ( true === $options['do_not_track'] )
+		return true;
+
+	return false;
+}
+
+function is_dnt_enabled() {
+	if ( false === apply_filters( 'jetpack_honor_dnt_header_for_stats', false ) )
```

Comment:
Could you add braces here as well, and add inline documentation for the filter, as it's done [here](https://github.com/perillamint/jetpack/blob/fab3e2b2665bbd42d24a7048f15ce30505fd2e70/modules/stats.php#L280)?